README.md

```diff
@@ -11,7 +11,7 @@
 [![GitHub issues-closed][closed-issues-img]][closed-issues-url]
 [![Average time to resolve an issue][ismaintained-resolve-img]][ismaintained-resolve-url]
 [![Percentage of issues still open][ismaintained-open-rate-img]][ismaintained-open-rate-url]
-[![GitHub commit rate](https://img.shields.io//github/commit-activity/m/Clinical-Genomics/scout)](https://GitHub.com/Clinical-Genomics/scout/pulse/)
```

Comment:
It's not found because there are two // after img.shields.io. But It makes sense to use a weekly button since the pulse default activity is the weekly one

Reply:
Mm, that is normally not an issue with URLs: could be for the cached image though! But actually also not generated if you check the link..

Reply:
Ah, ok, now the m also works. Maybe its just some caching issues. Anyway, w is good - lets see if the cache doesn't update soon.